messaging/WFLY-14133_configurable_bridge_call_timeout.adoc

```diff
@@ -0,0 +1,129 @@
+= Add call-timeout attribute to the Artemis core bridge resource
+:author:            Tomas Hofman
+:email:             thofman@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:          messaging,jms
+:idseparator:       -
+
+== Overview
+
+The Artemis broker defines a call-timeout attribute on the core bridge, which sets a time out for blocking calls to the
+server. Unless modified, the call-timeout defaults to 30 seconds.
```

Comment:
Yes, it can be modified also in the original version, but one has to use lower level API. Emmanuel Hugonnet recommended to extend the high level API instead of using the low level one, to avoid code duplication. I will update the paragraph.

Reply:
I reformulated the paragraph.